### PR TITLE
[SIBIU-2024] Add sibiu general info

### PR DESCRIPTION
**Pull request by mihai-satmarean**
Merge concerning the initial page status and program

### CODE REVIEW

**Comment by mihai-satmarean** [comment]
I wanted to create a draft request

**Comment by phrawzty** [comment]
> I wanted to create a draft request

👋 I have converted it into a draft request for you. 🙂

**Comment by mihai-satmarean** [comment]
Yes please Pe 14.05.2024, la 22:18, Adrian Moisey ***@***.***> a scris:﻿
Is this PR ready for review?

**Comment by phrawzty** [comment]
The event is scheduled for 6 June and you have neither a programme, nor a way to register to attend. Please update your PR with at least one of these two things—preferably both—as soon as possible.

**Comment by phrawzty** [comment]
Filenames must be be lowercase in order to conform to case-insensitive filesystem support (see [lint test failure](https://github.com/devopsdays/devopsdays-web/actions/runs/9291840850/job/25571318498?pr=13955#step:3:40464)); otherwise, it's looking good so far. :)

**Comment by valioprea** [comment]
Hey, @phrawzty - I'm helpping Mihai but unfortunately I don't have admin rights to this PR, so I can only comment here: solved most of the things, can this PR be reviewed and merged?

File: content/events/2024-sibiu/welcome.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+Title = "devopsdays Sibiu 2024"
+Type = "welcome"
+aliases = ["/events/2024-sibiu/"]
+Description = "devopsdays Sibiu 2024"
++++
+
+<div>
+
+  <div>
+    DevOps Days is a worldwide series of technical conferences covering topics of software development, IT infrastructure operations, and the intersection between them. The first time ever DevOps Days event in Romania is coming to Sibiu and it brings together international and local speakers, blending different experiences and topics.
```
Comment (phrawzty) [inline content/events/2024-sibiu/welcome.md:11]: DevOpsDays is one word, by convention. Not a blocker, but you may wish to adjust it in the future. :)

```suggestion
    DevOpsDays is a worldwide series of technical conferences covering topics of software development, IT infrastructure operations, and the intersection between them. The first time ever DevOpsDays event in Romania is coming to Sibiu and it brings together international and local speakers, blending different experiences and topics.
```

**Review by phrawzty — APPROVED** [review]

**Comment by phrawzty** [comment]
So glad to see that the first DevOpsDays in Romania is really going to happen! :D

**Pull request merged**